### DISCUSSION TITLE
Can't open Ver. 1.2 because of SQL

### DISCUSSION THREAD

**Discussion by JCEurovisionFan** [Maerciez/TheMillionaireGame#82] [Bugs and Errors!]
![image](https://github.com/user-attachments/assets/e64f5dc1-2de0-48e9-b284-63a1e14de6dc)
![image](https://github.com/user-attachments/assets/7da77cb1-63e7-48de-8ce8-1fa3340779a5)

The pictures above showed that Ver. 1.2 didn't work, despite installing SQL Express 2022. How can I make it work? I can't do SQL databases, I'm afraid.

**Comment by ristoryti** [comment]
At least for me, ticking the Use LocalDB box solved the issue. I haven't played a game yet but at least that's how I got the application to open!
